facia-tool/app/auth/PanDomainAuthActions.scala

```diff
@@ -42,8 +45,8 @@ trait PanDomainAuthActions extends AuthActions with Results {
 
   override lazy val domain: String = Configuration.faciatool.pandomainDomain
   override lazy val system: String = Configuration.faciatool.pandomainService
+  override def awsCredentialsProvider  = new AWSCredentialsProviderChain(
+    new STSAssumeRoleSessionCredentialsProvider(Configuration.faciatool.pandomainRoleArn, "workflow")
```

Comment:
Ah ok, I chose "workflow" because that's the account the of the role to assume, but I'm happy for this to be "fronts"